projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html

```diff
@@ -2,23 +2,26 @@
 <div *ngIf="error && !(error.callbackErrors || error.callbackWarnings || error.details)" class="error-summary"
      role="group" aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
   <h1 class="heading-h1 error-summary-heading" id="edit-case-event_error-summary-heading">
-    Something went wrong
+    {{'Something went wrong' | rpxTranslate}}
   </h1>
   <div class="govuk-error-summary__body" id="edit-case-event_error-summary-body">
-    <p>We're working to fix the problem. Try again shortly.</p>
-    <p><a href="get-help" target="_blank">Contact us</a> if you're still having problems.</p>
+    <p>{{"We're working to fix the problem. Try again shortly." | rpxTranslate}}</p>
+    <p>
+      <a href="get-help" target="_blank">
+        {{"Contact us" | rpxTranslate}}</a> {{"if you're still having problems." | rpxTranslate}}
+    </p>
   </div>
 </div>
 <!-- Callback error heading and error message to be displayed if there are specific error details -->
 <div *ngIf="error && error.details" class="error-summary" role="group"
      aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
   <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-heading">
-    The callback data failed validation
+    {{"The callback data failed validation" | rpxTranslate}}
```

Comment:
@DavidJayakumar - message text should be in single quotes

Reply:
done